plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -724,13 +724,13 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
         }
         Path targetPath = new Path(location);
         ensurePathExists(session, targetPath);
-        Path deltaLogDirectory = getTransactionLogDir(targetPath);
+        String deltaLogDirectory = getTransactionLogDir(location);
```

Comment:
Can we migrate `targetPath` two lines up as well?

Reply:
It's still used in code which will be removed in #16200. So I would wait for that PR to land and remove it during rebase.